docker/Dockerfile-multi-arch

```diff
@@ -0,0 +1,23 @@
+FROM golang:1.18.3 as build
+WORKDIR /workspace
+ENV GO111MODULE=on \
+    GOPROXY=https://goproxy.cn,direct \
+    BINARY=postgres-operator
+COPY . .
+RUN GOOS=linux GOARCH=$(go env GOARCH) CGO_ENABLED=0 go build -o build/linux/${BINARY} ./cmd/main.go
+
+FROM {base_image}
```

Comment:
I think you would need ARG BASE_IMAGE=alpine:3.15 because the official image has amd/arm support.
https://github.com/zalando/spilo/blob/master/postgres-appliance/Dockerfile#L18